### PR TITLE
typos and format fixed

### PR DESCRIPTION
**Pull request by DingqianZhou**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: website/_docs/install/configuration.md [hunk 1]
```diff
@@ -258,14 +258,14 @@ This section introduces Kylin data modeling and build related configuration.
 - `kylin.cube.aggrgroup.max-combination`: specifies the max combination number of aggregation groups. The default value is 32768.
 - `kylin.cube.aggrgroup.is-mandatory-only-valid`: whether to allow Cube contains only Base Cuboid. The default value is *FALSE*, set to *TRUE* when using Spark Cubing
 - `kylin.cube.rowkey.max-size`: specifies the maximum number of columns that can be set to Rowkeys. The default value is 63.
-- `kylin.cube.allow-appear-in-multiple-projects`: whether to allow a cube to appear in multiple projects
+- `kylin.cube.allow-appear-in-mulNotele-projects`: whether to allow a cube to appear in mulNotele projects
```
Comment (shaofengshi) [inline website/_docs/install/configuration.md]: What does "mulNotele" mean? This is a typo I think

File: website/_docs/install/configuration.md [hunk 2]
```diff
@@ -258,14 +258,14 @@ This section introduces Kylin data modeling and build related configuration.
...
 - `kylin.cube.gtscanrequest-serialization-level`: the default value is 1
 
 
 
 ### Cube Size Estimation {#cube-estimate}
 
-Both Kylin and HBase use compression when writing to disk, so Kylin will multiply its original size by the ratio to estimate the size of the cube.
+Both Kylin and HBase use compression when writing to disk, so Kylin will mulNotely its original size by the ratio to estimate the size of the cube.
```
Comment (shaofengshi) [inline website/_docs/install/configuration.md]: "mulNotely" is a typo I think

Comment (ian4hu) [reply website/_docs/install/configuration.md]: I guess it may caused by replacing `Tip` to `Note` like bellow.

Comment (DingqianZhou) [reply website/_docs/install/configuration.md]: My mistake!!!! Sorry for the carelessness!!!

Already corrected in the new commit in this pr.

**Comment by shaofengshi** [comment]
As this PR has conflictioni now, I merged it manually. Close this PR now

**Pull request closed without merging**